src/nvim/fold.c

```diff
@@ -539,12 +539,10 @@ int foldManualAllowed(int create)
   return FALSE;
 }
 
-/* foldCreate() {{{2 */
-/*
- * Create a fold from line "start" to line "end" (inclusive) in the current
- * window.
- */
-void foldCreate(linenr_T start, linenr_T end)
+// foldCreate() {{{2
+/// Create a fold from line "start" to line "end" (inclusive) in the current
+/// window.
```

Comment:
It's not "in the current window" anymore, but in the given window

Reply:
thanks for the review. This PR is a leap of faith as there are still missing pieces if you pass something else than `curwin` as a window. There is a bunch of work done on folds and this patch was lingering for the past ~4 years so I decided to go ahead. I am fixing things to provide an API right now.